### PR TITLE
Adding open speed test server

### PR DESCRIPTION
**Pull request by openspeedtest**
Thank you for your submission to the FreeNAS community plugins repository!

Please ensure the following guidelines are met when submitting a new Plugin.

1. Add the entry to INDEX in alphabetical order 
DONE!
2. Includes a new icon in the ./icon/ directory
DONE!
3. Ensure the submitted icon file is a valid PNG that is 128x128 in size
DONE!

### CODE REVIEW

File: INDEX [hunk 1]
```diff
@@ -216,6 +216,14 @@
         "official": false,
         "primary_pkg": "openvpn"
     },
+  "OpenSpeedTest-Server": {
+    "MANIFEST": "OpenSpeedTest-Server.json",
```
Comment (jsegaert) [inline INDEX:220]: Same remark here, use `openspeedtest-server.json` instead and also rename the corresponding file.

File: OpenSpeedTest-Server.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "OpenSpeedTest-Server",
+    "release": "11.3-RELEASE",
+    "artifact": "https://github.com/openspeedtest/iocage-plugin-OpenSpeedTest-Server.git",
+    "properties": {
+        "nat": 1,
+        "nat_forwards": "tcp(80:8080)"
```
Comment (jsegaert) [inline OpenSpeedTest-Server.json:7]: NGINX is listening to port 8080 in the jail and the FreeNAS host is already listening on port 80 for the UI, so `tcp(8080:8080)` would make more sense.

**Comment by openspeedtest** [comment]
Thanks for the feedback. 
I resolved the issues..

File: openspeedtest-server.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "release": "11.3-RELEASE",
+    "artifact": "https://github.com/openspeedtest/iocage-plugin-OpenSpeedTest-Server.git",
+    "properties": {
+        "nat": 1,
+        "nat_forwards": "tcp(8080:8080)"
+    },
+    "pkgs": [
+        "nginx"
+    ],
+    "packagesite": "http://pkg.FreeBSD.org/FreeBSD:11:amd64/latest",
```
Comment (kmoore134) [inline openspeedtest-server.json:12]: Could this FreeBSD:11:amd64 be changed to ${ABI} that way on 12.0 it fetches packages from 12 repo?

**Pull request merged**